### DISCUSSION TITLE
Plugin for Unreal Engine 4

### DISCUSSION THREAD

**Discussion by Informatyk1234** [JSBSim-Team/jsbsim#1106] [Q&A]
Is there any way to get this plugin for UE4? I need it because I use plugin for YAW VR in flight sim, which works only with UE4

**Comment by gallonmate** [comment]
You should ask the YAW VR makers to update their support for UE5. I'm not sure why their VR wouldn't work with UE5. In fact if you have a UE4 plugin for the YAW VR, it would probably be easier to update that plugin to work with UE5, than to make JSBSim plugin work with UE4.

That being said, I was planning to add UE4 support for this plugin but I simply do not have time at the moment.
